cmd/admin.go

```diff
@@ -611,19 +612,19 @@ func parseSMTPConfig(c *cli.Context, conf *smtp.Source) error {
 		conf.AllowedDomains = c.String("allowed-domains")
 	}
 	if c.IsSet("force-smtps") {
-		conf.ForceSMTPS = c.BoolT("force-smtps")
+		conf.ForceSMTPS = c.Bool("force-smtps")
```

Comment:
Wait, didn't the defaults change in this file?

Reply:
What default?

Reply:
If I'm not mistaken, `BoolT` returned `true` when no value has been set, correct?